tests/python_package_test/test_basic.py

```diff
@@ -328,3 +328,47 @@ def check_asserts(data):
         lgb_data.set_weight(sequence)
         lgb_data.set_init_score(sequence)
         check_asserts(lgb_data)
+
+    def test_trees_to_dataframe(self):
+
+        def _imptcs_to_numpy(X, impcts_dict):
+            cols = ['Column_' + str(i) for i in range(X.shape[1])]
+            imptcs = []
+            for col in cols:
+                try:
+                    imptcs.append(impcts_dict[col])
+                except KeyError:
+                    imptcs.append(0.)
+            return np.array(imptcs)
+
+        X, y = load_breast_cancer(True)
+        data = lgb.Dataset(X, label=y)
+        params = {
+            "objective": "binary",
+            "metric": "auc",
+            "min_data": 10,
+            "num_leaves": 15,
+            "verbose": -1,
+            "num_threads": 1,
```

Comment:
Why is it needed here?

Reply:
It's not. I thought these hyperparameters were standard across unit tests.

Reply:
Sorry for the confusion! My comment was addressed to `"num_threads": 1`. Anyway, other params are not necessary as well, and everything seems good now.